### PR TITLE
feat: cosmos mobile wallet support

### PR DESCRIPTION
**Pull request by codingki**

### CODE REVIEW

**Comment by ericHgorski** [comment]
Screenshot 2024-11-22 at 7 49 47 PM
encountered this error but dont have Sei active on mobile

**Comment by codingki** [comment]
> Screenshot 2024-11-22 at 7 49 47 PM encountered this error but dont have Sei active on mobile

This is resolved by showing a better error message

File: packages/widget/src/providers/WalletProviders.tsx [hunk 1]
```diff
@@ -1,15 +1,20 @@
...
-      <EVMProvider>
-        {children}
-      </EVMProvider>
-    </CosmosProvider>
+    <EVMProvider>
+      <QueryClientProvider client={queryClient} key={"skip-widget"}>
+        <CosmosProvider>
+          {children}
+        </CosmosProvider>
+      </QueryClientProvider>
```
Comment (toddkao) [inline packages/widget/src/providers/WalletProviders.tsx:17]: should `QueryClientProvider` be within `WalletProviders` ?

**Comment by ericHgorski** [comment]
I was able to connect keplr mobile to the app, but it got hung up on the validating gas/balance phase and I was never prompted with a TX signature.

**Review by ericHgorski — APPROVED** [review]

**Pull request merged**